src/Parser/Priest/Shadow/Modules/Items/AnundsSearedShackles.js

```diff
@@ -0,0 +1,55 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import { formatNumber } from 'common/format';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Analyzer from 'Parser/Core/Analyzer';
+
+const DAMAGE_INCREASE_PER_STACK = 0.03;
+const MAX_STACK = 50;
+
+class AnundsSearedShackles extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+  buffStacksSinceLastCast = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWrists(ITEMS.ANUNDS_SEARED_SHACKLES.id);
+  }
+
+  on_byPlayer_damage(event){
+    const spellID = event.ability.guid;
+    if(spellID === SPELLS.VOID_BOLT.id){
+      this.bonusDmg += event.amount - (event.amount / (1 + this.buffStacksSinceLastCast * DAMAGE_INCREASE_PER_STACK));
```

Comment:
The recommended approach for calculating these kind of damage bonuses is to use `calculateEffectiveDamage`. For example this is missing absorbed damage which more often than not is important in fights.

Reply:
Ah, I will get it changed.